image/templates/helm/shared/internal/scanner-v4-config-shape.yaml

```diff
@@ -0,0 +1,80 @@
+scannerV4:
+  disable: null # bool
```

Comment:
I figure having this high-level `disable` makes it easier to use than always checking `indexer.disable` and `matcher.disable`. In some of the configs, you can see when each `disable` is used.